swap/src/bin/asb.rs

```diff
@@ -173,6 +174,67 @@ async fn main() -> Result<()> {
 
             tracing::info!("Current balance: {}, {}", bitcoin_balance, monero_balance);
         }
+        Command::ManualRecovery(manual) => match manual {
+            ManualRecovery::Cancel {
+                cancel_params: RecoverCommandParams { swap_id, force },
+            } => {
+                let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
+
+                let (txid, _) =
+                    alice::cancel(swap_id, Arc::new(bitcoin_wallet), Arc::new(db), force).await??;
+
+                tracing::info!("Cancel transaction successfully published with id {}", txid);
+            }
+            ManualRecovery::Refund {
+                refund_params: RecoverCommandParams { swap_id, force },
+            } => {
+                let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
+                let monero_wallet = init_monero_wallet(&config, env_config).await?;
+
+                alice::refund(
+                    swap_id,
+                    Arc::new(bitcoin_wallet),
+                    Arc::new(monero_wallet),
+                    Arc::new(db),
+                    force,
+                )
+                .await??;
+
+                tracing::info!("Monero successfully refunded");
+            }
+            ManualRecovery::Punish {
+                punish_params: RecoverCommandParams { swap_id, force },
+            } => {
+                let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
+
+                let (txid, _) =
+                    alice::punish(swap_id, Arc::new(bitcoin_wallet), Arc::new(db), force).await??;
+
+                tracing::info!("Punish transaction successfully published with id {}", txid);
+            }
+            ManualRecovery::SafelyAbort { swap_id } => {
+                alice::safely_abort(swap_id, Arc::new(db)).await?;
+
+                tracing::info!("Swap safely aborted");
+            }
+            ManualRecovery::Redeem {
+                redeem_params: RecoverCommandParams { swap_id, force },
+                do_not_await_finality,
+            } => {
+                let bitcoin_wallet = init_bitcoin_wallet(&config, &seed, env_config).await?;
+
+                let (txid, _) = alice::redeem(
+                    swap_id,
+                    Arc::new(bitcoin_wallet),
+                    Arc::new(db),
+                    force,
+                    !do_not_await_finality,
```

Comment:
yeah... that is not optimal, I will see to model it as `enum` instead of the boolean value to make it more readable.
I will see to keep this as a `bool` flag on the structopt level though.